docs/api/index.rst

```diff
@@ -1,16 +1,64 @@
 .. _api-endpoints:
 
-The HTTP APIs
-#############
+HTTP endpoints
+##############
+
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| Method   | URI                                                                                          | Description                                             |
++==========+==============================================================================================+=========================================================+
+| `GET`    | :ref:`/ <api-utilities>`                                                                     | :ref:`Information about the running instance            |
+|          |                                                                                              | <api-utilities>`                                        |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `POST`   | :ref:`/batch <batch>`                                                                        | :ref:`Send multiple operations in one request <batch>`  |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `GET`    | :ref:`/__heartbeat__ <api-utilities>`                                                        | :ref:`Return the status of dependent services           |
+|          |                                                                                              | <api-utilities>`                                        |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `PUT`    | :ref:`/buckets/(bucket_id) <bucket-put>`                                                     | :ref:`Create or replaces a bucket <bucket-put>`         |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `GET`    | :ref:`/buckets/(bucket_id) <bucket-get>`                                                     | :ref:`Retrieve an existing bucket <bucket-get>`         |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
+| `DELETE` | :ref:`/buckets/(bucket_id) <bucket-delete>`                                                  | :ref:`Delete a bucket <bucket-delete>`                  |
++----------+----------------------------------------------------------------------------------------------+---------------------------------------------------------+
```

Comment:
Good catch, I added them (and actually the collection get on groups was missing — now fixed)